llpc/tool/llpcPipelineBuilder.cpp

```diff
@@ -84,19 +84,24 @@ bool PipelineBuilder::isGraphicsPipeline() const {
 //
 // @returns : Result::Success on success, other status on failure.
 Result PipelineBuilder::build() {
-  BinaryData pipeline = {};
+  llvm::SmallVector<BinaryData, 1> pipelines;
```

Comment:
Could you add a comment explaining why we need more than one BinaryData? This is not clear from just the code.

Reply:
nit: no need to spell out the llvm namespace (`llvm::`) in `.cpp` files